cometd-java/cometd-java-benchmark/cometd-java-benchmark-server/src/main/java/org/cometd/benchmark/server/CometDLoadServer.java

```diff
@@ -515,7 +517,13 @@ private void formatStackFrames(StackTraceElement[] stackFrames, StringBuilder bu
     }
 
     private static class RequestLatencyHandler extends HandlerWrapper implements MeasureConverter {
-        private final AtomicHistogram histogram = new AtomicHistogram(TimeUnit.MINUTES.toNanos(1), 3);
+        private final Collection<Histogram> allHistograms = new CopyOnWriteArrayList<>();
+        private final ThreadLocal<Histogram> histogramTl = ThreadLocal.withInitial(() ->
+        {
```

Comment:
Rename variable to just `histogram` and brace at EOL.

Reply:
Ok.